cogs/tags/tags.py

```diff
@@ -196,7 +196,7 @@ async def syncAllowedRoles(self):
             async with self.configV3.guild(guild).tiers() as tiers:
                 self.allowed_roles[guildId] = set(tiers.keys())
         self.logger.debug(
-            "Roles allowed to create commands: %s", ", ".join(self.allowed_roles.keys())
+            "Roles allowed to create commands: %s", ", ".join(map(str, self.allowed_roles.keys()))
         )
```

Comment:
Just realized, this was printing the wrong thing before (it was printing only the server ID and not the role IDs themselves; wrong indent block). Can we do something like:
```suggestion
            self.logger.debug(
                "Roles allowed to create commands on %s: %s", guildId,
                ", ".join(tiers)
            )
```

Reply:
Yep. Should be quick.
Might as well apply Python's `f`-string there.

The block should be indented twice. Also `tiers` could be a `list` or a `dict`, so applying `str` should be fine, but you might want to see if anything extra other than its keys that should be printed out in case of `dict`.

Reply:
Ah, in this case, don't use f-string. The reason is because f-string will do string substitution even if log is not printed (logger recommends %s for log string substitution, we kind of broke the rule with .join lol). 

Yeah, tiers is a dict, key is role ID, value is max number of tags allowed.

Reply:
Oh that's new. I've just researched a bit and yes logging docs do recommend `%` format for optimization.